src/DefaultValidatorLocator.cs

```diff
@@ -22,7 +22,7 @@ private IValidator FindValidator(Type type)
             var fullType = CreateValidatorType(type);
 
             var available = this.validators
-                .Where(validator => fullType.GetTypeInfo().IsAssignableFrom(validator.GetType()))
+                .Where(validator => fullType.GetTypeInfo().IsInstanceOfType(validator))
```

Comment:
Did you run tests for this. I saw Rider suggest this but I'm not sure it does what I want

Reply:
Did I run tests before sending a PR... it's like you don't even know me. (Yes, the tests pass)

Reply:
I need to get a CI build setup on this repo. That's your next PR 😄

Reply:
AppVeyor -> New Project, right?

Reply:
If you want windows. There's others though https://github.com/marketplace/category/continuous-integration

Reply:
Unfortunately for you, it will have to be the repo admin that does it. Unlucky.